.travis.yml

```diff
@@ -7,12 +7,6 @@ os: osx
 osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_NPY=110  CONDA_PY=34
-    - CONDA_NPY=111  CONDA_PY=34
-    - CONDA_NPY=110  CONDA_PY=35
-    - CONDA_NPY=111  CONDA_PY=35
```

Comment:
Hmm...this shouldn't be removed. Could you please edit your `.condarc` to include this...

```yaml
channels:
  - conda-forge
  - defaults
```

Also comment out any other `channels` section. Then please try to re-render again.